src/components/TopBar.js

```diff
@@ -16,6 +16,10 @@ export const styles = {
   icon: {
     color: topBarTextColor,
   },
+  term: {
+    color: topBarTextColor,
+    marginLeft: 'auto',
```

Comment:
Instead of doing it with a left margin, let's use the power of flexbox. If you haven't used flex before, you're gonna wanna bookmark this: https://css-tricks.com/snippets/css/a-guide-to-flexbox/ Give that a quick perusal to get a sense of how it works.

Flexbox is an incredibly powerful and useful css solution that gets used quite often. In this case, we want two "things" (the icon + the Serif.nu title is one, the and the term indicator is the other) to be aligned on opposite sides. We'll need to do two things.

1. You'll have to throw a div around the already existing `IconButton` and Serif.nu `Typography` components, so that flexbox knows to tread both of those as one unit.
2. Then you'll have to provide a className to the `Toolbar` component that contains the styles
```
display: 'flex',
justifyContent: 'space-between',
```
This way, we make the parent container a flexbox container and tell flexbox to space our two items on either end. Now you no longer need the `term` className.

Reply:
oof I suggested marginLeft, this is a learning opportunity for me too